src/freenet/pluginmanager/PluginManager.java

```diff
@@ -1001,7 +1001,7 @@ public List<OfficialPluginDescription> findAvailablePlugins() {
 	}
 
 	public OfficialPluginDescription isOfficialPlugin(String name) {
-		if((name == null) || (name.trim().length() == 0))
+		if((name == null) || (name.trim().isEmpty()))
```

Comment:
Parens.

Reply:
Fixed in ed98d8c127320b71b507f1524cce60e0d5efa356